test/jest/matchers.js

```diff
@@ -0,0 +1,38 @@
+import { shallow, mount } from 'enzyme';
```

Comment:
The code in this file is copied from `enzyme-matchers` (https://github.com/enzymejs/enzyme-matchers/blob/master/packages/enzyme-matchers/src/assertions/toMatchElement.js and https://github.com/enzymejs/enzyme-matchers/blob/master/packages/enzyme-matchers/src/utils/isShallowWrapper.js). The reason I copied it is because `jest-enyzme` [relies on an old version of jsdom](https://github.com/enzymejs/enzyme-matchers/issues/351) and including it would break a bunch of tests. Based on the [inactivity of this repo](https://github.com/enzymejs/enzyme-matchers/issues/353) it doesn't seem like this issue will be addressed any time soon. 

In terms of why I chose to add this matcher, I looked through some existing test files and it seems like in most cases we merely check that a component contains other components. This did not work for my case as I needed to check the contents of the ContactList component (albeit shallowly) and thus confirm the sort order of children elements. Happy to change the test below to accommodate another style, however.